composer.json

```diff
@@ -45,7 +45,7 @@
     "ext-json": "*",
     "ext-libxml": "*",
     "ext-mbstring": "*",
-    "ext-psr": "^0.6",
+    "ext-psr": "^0.7",
```

Comment:
I am going to revert this, as soon as #14297 is merged.

Reply:
@zsilbi Fixed in the 4.0.x branch. Could you please rebase

Reply:
Rebased.